controllers/controller.js

```diff
@@ -19,3 +23,15 @@ exports.getArticles = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getArticleById = (req, res, next) => {
+  const { id } = req.params;
+  selectArticlesById(id)
```

Comment:
you've called this function get Articles by id but I recon when we give an Id we probably only want one article to be returned, I would suggest renaming this function and have it return the single article, this would stop you needing to do any data manipulation in the controller too